app/views/properties/show.html.erb

```diff
@@ -90,24 +90,34 @@
         </h5>
         <hr>
         <div class="row small-up-2">
-            <%= Amenity.find_each do |amenity| %>
-            <% if @property.amenities.include?(amenity) %>
-            <div class="column">
-                <label>
-                    <%= amenity.name %>
-                </label>
-              </div>
-              <% else %>
-              <div class="column">
-                  <label>
-                      <del><%= amenity.name %></del>
-                  </label>
+            <div class="js-amenities">
+                <% Amenity
```

Comment:
Should probably be one line and all you need is `Amenity.find_each do |amenity|`. Find each in an enumerator just the same as each.